content/blog/2021/volunteering.md

```diff
@@ -0,0 +1,102 @@
+Title: Help us PyCon India 2021 even more amazing
+Date: 2021-02-26 17:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2021
+Slug: 2021-call-for-volunteers
+Summary: Want to give back to the Python community? Here’s an awesome opportunity for you to do it with the year's premier Python conference, **PyCon India 2021**.
+
+[PyCon India](https://in.pycon.org/2021/) is a grass-roots, volunteer-driven conference. What that means is every year for the past 13 years, Python communities across the nation have come together to help plan, organize, manage, and run the conference. The conference is operated entirely by volunteers from planning to finding the speakers to managing call for proposals to handling the mega event successfully without fail.
+
+## Things you should know before volunteering
+
+Before you volunteer to help the community make the conference even more amazing, there are a few things you should know:
+
+* There is a lot of work that goes into a conference, and all of it is done by the volunteers. So, naturally, volunteering requires a lot of time and effort, and some activities like running outreach and social media, building content, and finding speakers, a lot more than others. So, make sure you can allocate sufficient time to your volunteering efforts.
+
+
+* If you are engaged in a volunteering activity while a session is going on, you might not be able to attend that session. If someone needs your assistance with a task, you need to be there. So, plan your conference schedule accordingly.
+
+
+* Volunteering does not provide you a free way into the event. At PyCon, everyone needs to buy a ticket, even the organizers. We volunteer because we love Python and the community, and want to make this event great, not to get a free pass to the conference.
```

Comment:
PyCon -> PyCon India

Reply:
We volunteer because we love Python and the community, -> We as a volunteer love the Python Programming language and the community.

Reply:
```suggestion
* Volunteering does not provide you a free way into the event. At PyCon India, everyone needs to buy a ticket, even the organizers. We volunteer because we love the Python programming language and the community, and want to make this event great, not to get a free pass to the conference.
```

Wait, the 2nd one doesn't make sense with the rest of the sentence. I've just added "programming language" for now.